RELEASE_PROCESS.md

```diff
@@ -32,6 +32,13 @@ In order to release, you will need the following accounts/permissions:
 
 - [ ] Announce your intention to drive the release to #cft-announce and #chef-server on slack.
 
+### Getting the build to be released into current
+
+- Update the version file to the version to be released.
+- Update the release notes with the version, date and context.
```

Comment:
This duplicates a step down in `Preparing for the release`. I think that we can delete that section completely now that we have this new section.